metroplex.sh

```diff
@@ -1,4 +1,3 @@
-export DISPLAY=:0.0 # just DISPLAY=:0.0 without export is not enough
```

Comment:
This one is part of our default script. Now, since everything is happening in  docker, it makes sense to remove it.